cray_product_catalog/catalog_update.py

```diff
@@ -232,6 +249,20 @@ def update_config_map(data, name, namespace):
             else:
                 LOGGER.exception("Error calling replace_namespaced_config_map")
 
+def split_catalog_data(data):
+    """Split the passed data into data needed by main and product config map"""
+    main_cm_data = {}
```

Comment:
We can directly create dictionary with dict comprehension and return:

return {k:data[k] for k in set(data.keys())-PRODUCT_CM_FIELDS}, {k:data[k] for k in PRODUCT_CM_FIELDS}